robotcontainer.py

```diff
@@ -110,5 +153,16 @@ def configureButtonBindings(self):
             *self.operatorInterface.driveToTargetControl
         ).whenHeld(DriveToTarget(self.drive, constants.kAutoTargetOffset))
 
+        commands2.button.JoystickButton(
+            *self.operatorInterface.autoBallIntakeControl
+        ).whenHeld(AutoBallIntake(self.drive, self.intake))
+
+        commands2.button.JoystickButton(*self.operatorInterface.shootBall).whenHeld(
+            ShootBall(self.indexer)
+        )
```

Comment:
Why not link this .whenHeld and the next whenRelease on the same button together, like on line 104?